### PR TITLE
Task 1 add initial rails config

### PR DESCRIPTION
**Pull request by Luke-kb**
## Description
added initial rails config
moved pull_request_template to .github directory

## Breakdown of Commits
Commit 1 : 0a42f83a7ca90fe60afc96980659626aa85bf403
- added initial rails config files
- pull_request_template in root directory
 
Commit 2 : b3749b51b138aa8086bc7615fc72a739225be352
- moved pull_request_template to .github directory

#### TODO Checklist
* [ ] Keep Your Pull Requests Small
* [ ] Adhere to Ruby/Rails/Rspec Style guides
* [ ] 

  
#### Screenshot (If applicable)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+source 'https://rubygems.org'
+
+gem 'rails', '3.2.0'
+
+# Bundle edge Rails instead:
+# gem 'rails', :git => 'git://github.com/rails/rails.git'
+
+gem 'mysql2'
```
Comment (jbat) [inline Gemfile:8]: Might be a good idea to go with the same version so the functionality matches
The JRPlus project uses this version
gem 'mysql2', '0.3.20'

**Review by jbat — APPROVED** [review]
lg +1

File: Gemfile (hunk 1 above)
Comment (Luke-kb) [reply Gemfile]: updated to v0.4.10 instead as gem native extension build error persisted for v0.3.20

**Comment by Luke-kb** [comment]
downgraded mysql2 to 0.4.10 to match JRPlus project

**Pull request merged**